lib/tokenizer/rainscriptTokenizer.js

```diff
@@ -0,0 +1,72 @@
+import Lexer from 'lex';
+import { _ } from 'underscore';
```

Comment:
I highly recommend lodash over underscore

Reply:
After quick googling, lodash indeed looks superior. Will replace all underscore to lodash in the next commit.